scripts/lint_jjb.py

```diff
@@ -155,6 +168,134 @@ def check_retention(job, in_file):
     return return_value
 
 
+# Ensure all jobs have a valid timed triggers entry
+# Structure:
+#   job:
+#     triggers:
+#      - timed: "@daily"
+def check_timed_trigger_value(trigger):
+
+    # If the trigger provided is a NoneType,
+    # then the job has no value, so pass it.
+    if type(trigger) == 'NoneType':
```

Comment:
Not sure this works:

```
>>> lolz = None
>>> type(lolz)
<type 'NoneType'>
>>> lolz == 'NoneType'
False
>>>
```

Reply:
Take a look at the function `isinstance` for testing the type of an object.

Reply:
That should be ``NoneType`` and not ``Nonetype``. :)

Reply:
@git-harry Done, thanks.